Index: sputnik-ci.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- sputnik-ci.py	(revision b7ce92ee5b6d4d768c431d2865e04081c9181227)
+++ sputnik-ci.py	(revision )
@@ -58,6 +58,9 @@
             checkstyle_url = "http://sputnik.touk.pl/conf/" + get_env("TRAVIS_REPO_SLUG") + "/checkstyle?key=" + get_env("api_key")
             download_file(checkstyle_url, "checkstyle.xml")
 
+            scalastyle_url = "http://sputnik.touk.pl/conf/" + get_env("TRAVIS_REPO_SLUG") + "/scalastyle?key=" + get_env("api_key")
+            download_file(checkstyle_url, "scalastyle.xml")
+
         sputnik_jar_url = "http://repo1.maven.org/maven2/pl/touk/sputnik/1.6.0/sputnik-1.6.0-all.jar"
         download_file(sputnik_jar_url, "sputnik.jar")
 
